tests/test_download_fluka.py

```diff
@@ -0,0 +1,64 @@
+import os
+import sys
+import pytest
+import logging
+import subprocess
+from pathlib import Path
+from yaptide.admin.simulator_storage import download_fluka_from_s3
+
+def check_if_environment_variables_set() -> bool:
+    """Check if environment variables are set"""
+    result = True
+    for var_name in ['S3_ENDPOINT', 'S3_ACCESS_KEY', 'S3_SECRET_KEY', 'S3_FLUKA_BUCKET', 'S3_FLUKA_KEY']:
+        if var_name not in os.environ:
+            logging.error('variable %s not set', var_name)
+            result = False
+    return result
```

Comment:
This exists in couple of tests now. Maybe this method could get some arguments and moved to one place (conftest.py?) to avoid code duplication ?

Reply:
Please check if it is ok now